### PR TITLE
Aframe/master

### PR DESCRIPTION
**Pull request by Utopiah**
basic sphere setup like in our PoC

### CODE REVIEW

File: aframe.html [hunk 1]
```diff
@@ -17,6 +17,8 @@
 AFRAME.registerComponent('console', {
   init: function () {
     let url = localStorage['url'] || 'http://gateway.local:8080';
+    if (!localStorage['url'])
+	    console.warn( "Gateway token unset. Visit your gateway Settings -> Developer -> Create local authorization" )
```
Comment (rzr) [inline aframe.html]: ok, 
maybe move upper line and remove the ||

for indentation is there any rule ? I tend to use 2 spaces ?

**Review by rzr — CHANGES_REQUESTED** [review]
maybe single commits are preferred ?

File: js/aframe/index.js [hunk 2]
```diff
@@ -8,15 +8,38 @@
...
+
+  // https://github.com/gmarty/aframe-ui-components
+  // https://github.com/caseyyee/aframe-ui-widgets
+  // https://github.com/rdub80/aframe-gui
+  var newEl = document.createElement( "a-sphere" )
+  newEl.setAttribute( "position", "0 " + ( viewer.y += 0.3 ) + " 0" )
+  newEl.setAttribute( "radius", "0.1" )
+  newEl.setAttribute( "color", "lightblue" )
+  var readwrite = true
+  if ( readwrite ){
```
Comment (rzr) [inline js/aframe/index.js:26]: maybe we need to define a style and use same everywhere

File: js/aframe/index.js [hunk 3]
```diff
@@ -8,15 +8,38 @@
...
 var viewer = {};
+viewer.y = 0;
 
 viewer.createViewUnknown = function(model)
 {
   let view;
   var value = viewer.el.getAttribute('text', value).value;
   if (model.type) {
-    value += "\nTODO: " +  model.type;
+    value += `\n${model.name} (${model.type})`;
```
Comment (rzr) [inline js/aframe/index.js]: maybe this part should be removed if we create elements ?

**Review by rzr — CHANGES_REQUESTED** [review]
changes are fine, but I would have start to implements binary resources because unknown types are not supposed to be implemented if we implements all other type ?

File: js/aframe/index.js (hunk 2 above)
Comment (Utopiah) [reply js/aframe/index.js:26]: Your choice, I have no preference

**Pull request closed without merging**